packages/gestalt/src/SelectList.js

```diff
@@ -162,7 +164,11 @@ export default class SelectList extends Component<Props, State> {
               </option>
             )}
             {options.map(option => (
-              <option key={option.value} value={option.value}>
+              <option
+                key={option.value}
+                value={option.value}
+                disabled={option.disabled || false}
```

Comment:
do we need `|| false`, the default is false already? Is Flow complaining without it?

Reply:
Looks like it wasn't needed! Updated accordingly--thanks!

Reply:
Thanks